pkgs/top-level/all-packages.nix

```diff
@@ -35811,7 +35811,7 @@ with pkgs;
     xwaylandSupport = false;
   };
 
-  chatterino2 = libsForQt5.callPackage ../applications/networking/instant-messengers/chatterino2 {
+  chatterino2 = qt6Packages.callPackage ../applications/networking/instant-messengers/chatterino2 {
```

Comment:
Since you've prefixed with `qt6`, I'd recommend you to use `callPackage` directly so it is more compatible with `by-name` migration. See https://nixos.org/manual/nixpkgs/unstable/#sec-language-qt